system_parameters:
  dt: 0.00001
  prediction_horizon: 20
  sampling_number: 10000

distribution_parameters:
  mean: 0
  variance: 0.1

inverse_temperature: 1.0

n_control: 9
n_state: 5

P_Macro: 801.325  # 700 + 101.325 kPa

Pump_parameters:
  Orifice_diameter: 1.6 # mm

Valve_parameters:
  Orifice_diameter: 1.6 # mm

Ejector_parameters:
  Orifice_diameter: 10  # mm
  dt : 0.00001

Actuator_parameters:
  length_of_neg_octagon: 20 # mm
  radius_of_load_shaft: 5 # mm
  External_load: 82 # kg
  L_max: 20 # mm
  dt : 0.00001
  positive_diameter : 100 # mm
  positive_length : 50 # mm
  v_tot : 


Tank_parameters:
  volume: 100             # mL

simulation_parameters:
  frequency: 1000000

weights:
  A: 1
